.gitignore

```diff
@@ -2,6 +2,7 @@
 
 *.bundle.js
 *.bundle.js.map
+/static/*
```

Comment:
Might need to re-think the static setup as it's being treated partly as a `/dist` folder by the Webpack setup and Node tools, and you theoretically shouldn't ever put stuff in there manually. Fine for now though.